File: src/C#/BehavioralPatterns/Mediator/CommunicationHub/Mediators/BroadcastMediator.cs
using CommunicationHub.Participants;
using System;
using System.Collections.Generic;
using System.Text;

namespace CommunicationHub.Mediators
{
    public class BroadcastMediator : IMediator
    {
        public List<Participant> Participants { get; set; } = new List<Participant>();

        public void Notify(Participant sender, object senderArgs)
        {
            Participants.ForEach(p => p.Receive(sender, senderArgs));
        }
    }
}

File: src/PHP/BehavioralPatterns/Mediator/CommunicationHub/Mediators/BroadcastMediator.php
<?php

namespace BehavioralPatterns\Mediator\CommunicationHub\Mediators;

class BroadcastMediator implements IMediator
{
    public array $participants = [];

    public function __construct()
    {
        $this->participants = [];
    }

    public function notify($sender, $senderArgs): void
    {
        foreach ($this->participants as $participant) {
            $participant->receive($sender, $senderArgs);
        }
    }
}